samples/tab-personal-quickstart/ts/Manifest/manifest.json

```diff
@@ -1,14 +1,14 @@
 {
-  "$schema": "https://developer.microsoft.com/en-us/json-schemas/teams/v1.12/MicrosoftTeams.schema.json",
-  "manifestVersion": "1.12",
+  "$schema": "https://developer.microsoft.com/en-us/json-schemas/teams/v1.13/MicrosoftTeams.schema.json",
+  "manifestVersion": "1.13",
   "version": "1.0.0",
-  "id": "{{Microsoft-App-Id}}",
+  "id": "${{TEAMS_APP_ID}}",
   "packageName": "com.microsoft.teams.extension",
   "developer": {
     "name": "Teams App, Inc.",
-    "websiteUrl": "https://{{domain-name}}",
-    "privacyUrl": "https://{{domain-name}}/privacy",
-    "termsOfUseUrl": "https://{{domain-name}}/termsofuse"
+    "websiteUrl": "https://www.microsoft.com",
```

Comment:
same here. Is it by design to use www.microsoft.com, or better to use ${{TAB_DOMAIN}}?

Reply:
Fixed